swap_meet/item.py

```diff
@@ -1,2 +1,35 @@
 class Item:
-    pass
+    """
+    Class Item with method: condition_description
+    """
+    # class constructor
+    def __init__(self, category="", condition=0):
+        self.category = category
+        self.condition = condition
+
+    # stringify instances of class Item
+    def __str__(self):
+        return "Hello World!"
+
+    # returns description of Item condition given condition(int) between 0 and 5
+    def condition_description(self):
```

Comment:
Nice use of a try-except.  I do think that this method shouldn't print if there's a problem instead either throw an error or return something to indicate a problem.